L59-core-allow-cppstdlib.md

```diff
@@ -0,0 +1,97 @@
+Allow C++ standard library in gRPC Core Library
+----
+* Author(s): veblush
+* Approver: a11r
+* Status: Draft
+* Implemented in: n/a
+* Last updated: August 19, 2019
+* Discussion at: https://groups.google.com/forum/#!topic/grpc-io/umQyyXmNr2w
+
+## Abstract
+
+Allow C++ Standard Library to be used in the gRPC Core library.
+
+## Background
+
+Since C++ was allowed to write gRPC, more and more code has been written in C++.
```

Comment:
Can we use language that focuses more on what the motivation and rationale (dev productivity etc.) rather than describing dry facts of what happened/is going to happen?